Scripts/Services/VisualRecognition/V4/VisualRecognitionService.cs

```diff
@@ -72,6 +72,7 @@ public bool DisableSslVerification
         /// <param name="authenticator">The service authenticator.</param>
         public VisualRecognitionService(string versionDate, Authenticator authenticator) : base(versionDate, authenticator, serviceId)
         {
+            Log.Warning("VisualRecognition Deprecation Warning:", "On 1 December 2021, Visual Recognition will no longer be available. For more information, see https://github.com/watson-developer-cloud/unity-sdk/tree/master#visual-recognition-deprecation.");
```

Comment:
does `[Obsolete("message"]` work on the class? see https://github.com/watson-developer-cloud/dotnet-standard-sdk/blob/e8f160aac7c3d84e2279e30d4fa092ca1591a90b/src/IBM.Watson.PersonalityInsights.v3/PersonalityInsightsService.cs

Reply:
Let me try